### ISSUE TITLE
macOS apps crash when built with 22.1.0

### ISSUE THREAD

**Issue by lutzroeder** [electron-userland/electron-builder#4390]
* **Version**: 22.1.0
* **Target**: macOS

**Actual**:
`.app` crashes after opening the main window.

**Repro steps**:
1. Download and unzip [electron-builder-4390.zip](https://github.com/electron-userland/electron-builder/files/3838608/electron-builder-4390.zip)
2. `cd electron-builder-4390/21.2.0`
3. `npm install`
4. `npx electron .` (works)
5. `npx electron-builder --mac`
6. Launch `./dist/mac/electron-builder-4390.app/Contents/MacOS/electron-builder-4390`

**Log**
```
~/: ./dist/mac/electron-builder-4390.app/Contents/MacOS/electron-builder-4390 
[59493:1103/231235.221684:FATAL:gpu_data_manager_impl_private.cc(962)] The display compositor is frequently crashing. Goodbye.
zsh: trace trap  ./dist/mac/electron-builder-4390.app/Contents/MacOS/electron-builder-4390
```

**Comment by zlab** [comment]
the same issue

**Comment by DenysVuika** [comment]
I can confirm that the apps won't start at all. I have managed to get the electron working on macOS by running `electron-builder --dir -c.mac.identity=null` but the app content is empty.

**Comment by idanwork** [comment]
Same here on macOS 10.14.16

**Comment by lutzroeder** [comment]
@devlar can you provide some pointers to investigate or is it possible to revert recent changes that caused this to happen?

**Comment by lutzroeder** [comment]
#4244 is the change causing this to happen.

**Comment by lutzroeder** [comment]
Fixed in 22.2.0

**Comment by ahmadwaliesipick** [comment]
i am using electron-builder 22.2.0 and electron 6.0.10 i have same error
[60389:0116/153159.902739:FATAL:gpu_data_manager_impl_private.cc(894)] The display compositor is frequently crashing. Goodbye.
